styles.css

```diff
@@ -46,3 +46,12 @@
     transform: scale3d(1.1, 1.1, 1);
   }
 }
+
+.input {
+  min-width: 280px
```

Comment:
https://tailwindcss.com/docs/min-width

Can you use tailwind convention style and delete this if unnecessary ?